utils/frame/benchmarking-cli/Cargo.toml

```diff
@@ -55,6 +55,6 @@ sp-trie = { version = "7.0.0", path = "../../../primitives/trie" }
 gethostname = "0.2.3"
 
 [features]
-default = ["rocksdb"]
+default = ["paritydb"]
```

Comment:
Rocksdb is already an optional feature. Not sure why you are then changing this. This means that you could disable rocksdb already.

Reply:
thanks, i'm brushing up on the Rust documentation :)